### ISSUE TITLE
xCode Archive fails

### ISSUE THREAD

**Issue by ChechoCZ** [realm/realm-js#3076]
### What are you trying to achieve? 

Publish my app to app store

### What did you expect to happen? 

Archive my app using xcode.

### What happened instead?
Build failed 

I have installed realm in my React Native project and it works well in Android. Although, in iOS, despite I can run the application in the simulator, when I proceed to archive it to publish to the app store, it throws this error:

```
Aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on iOS 11 or newer
```
This happen in different files: alloc.hpp to mention one of them.

I tried changing the iOS target version but it didn't work.

Any suggestions?

Thanks in advance.

### What are steps we can follow to reproduce this issue? 

Add realm as a dependency and try yo archive the project using xcode.

## Version of Realm and Tooling

- Realm JS SDK Version: 10.0.0-beta.7?
- Node: 12.2.0
- React Native: 0.62
- Client OS & Version: MacOS Catalina 10.15.5

**Comment by kraenhansen** [comment]
Hi @ChechoCZ thanks for reporting this, I'll have a look.

**Comment by kevin-lii** [comment]
Any update on this? @kraenhansen

**Comment by kneth** [comment]
Similar to

* https://github.com/facebook/rocksdb/issues/4064
* https://github.com/Microsoft/cpprestsdk/issues/1000

We might have to investigate our compiler options.

**Comment by aliozinan** [comment]
same problem here. @kraenhansen any updates?

**Comment by steffenagger** [comment]
This will be included in the upcoming `10.0.0-X` release.
See solution here: https://github.com/realm/realm-js/pull/3260
